### ISSUE TITLE
LAB04 - Error in number of fault domains

### ISSUE THREAD

**Issue by renaudwangler** [MicrosoftLearning/20533-ImplementingMicrosoftAzureInfrastructureSolutions#289]
Hi.
I think there is a problem with the lab 4 : the LAK states we should see 3 fault domain in the Availability set.
After the Add-20533Denvironment, we see only 2 (default value) fault domains.
The file e:\labfiles\lab04\starter\templates\azuredeploy.parameters.json explicitely invokes 3 fault domains but I can't see any call to this file in the setup (Just a guess trying to find the cause).
We should correct this or change the LAK file :)

**Comment by itbland** [comment]
> Fault domains define the group of virtual machines that share a common power source and network switch. By default, the virtual machines configured within your availability set are separated across up to three fault domains for Resource Manager deployments (two fault domains for Classic).

[https://docs.microsoft.com/en-us/azure/virtual-machines/windows/manage-availability](url)

**Comment by polichtm** [comment]
The maximum number of fault domains is region dependent - some regions support only 2 fault domains. Pls refer to https://docs.microsoft.com/en-us/azure/virtual-machines/windows/manage-availability

**Comment by renaudwangler** [comment]
Thanks for your answers. The issue we have here is a real lab-oriented issue : My student and I used the "West Europe" region (I know : you guessed it because of my poor English : I'm French ^^ ) wich permits 3 fault domains and we used the last generation of PS Module to provision the lab4. The Add-20533DEnvironment command provisioned 2 fault domains and the LAK states we should have 3 :)
(we did not create the availability set bu relied on the one created by the PS module.
